share/lib/common.sh

```diff
@@ -27,10 +27,10 @@ service_action() {
   service="$1"
   action="$2"
 
-  if [ -x /usr/sbin/service ]; then
-    CMD="service ${service} ${action}"
-  elif [ -x /etc/init.d/${service} ]; then
+  if [ -x /etc/init.d/${service} ]; then
```

Comment:
I'd still recommend for mid/long term to write a proper systemd service unit for OS versions that run systemd.